openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -106,6 +109,13 @@ define OPENPOWER_PNOR_INSTALL_IMAGES_CMDS
             dd if=$(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_FILENAME) of=$(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_UPDATE_FILENAME) bs=32M count=1; \
             $(INSTALL) $(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_UPDATE_FILENAME) $(BINARIES_DIR); \
         fi
+
+        # If this is a VPNOR system, run the generate-squashfs command and
+        # create a tarball
+        if [ "$(BR2_BUILD_VPNOR)" == "y" ]; then \
+            PATH=$(PATH):$(HOST_DIR)/usr/bin $(HOST_DIR)/usr/bin/generate-squashfs -f $(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_FILENAME).squashfs.tar $(STAGING_DIR)/pnor/$(BR2_OPENPOWER_PNOR_FILENAME); \
```

Comment:
So, I do not think that the files -> pnor -> files -> squashfs workflow is a sane one to have in the medium to long term.

In fact, the way we generate a PNOR now is needlessly complex, and having this extra step simply adds complexity. I don't like complexity :)

In the near future, it's *highly* likely we redo how we generate the PNOR image, and generate it straight from a CSV description and files, likely using https://github.com/open-power/skiboot/commit/efb9ae7fb411d8e0b194b9e345696f868e5e094c#diff-ed4cef76bf7974b27d75e3d6fc8ab15e tool and format.

While that won't immediately break constructing the squashfs, I'd like to see the same input be used to generate it, without having to go via the PNOR step itself (if only because then it needlessly serialises build steps)